qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -946,12 +1013,11 @@ def _copy_metadata(
                 # inherits an abstract class.
                 extra_kwargs = {}
                 for key, val in orig_kwargs.items():
-                    if key not in ["job", "job_done_callback", "timeout"]:
+                    if key not in ["jobs", "timeout"]:
                         extra_kwargs[key] = val
 
                 new_instance.add_data(
-                    data=orig_kwargs["job"],
-                    post_processing_callback=orig_kwargs["job_done_callback"],
+                    data=orig_kwargs["jobs"],
```

Comment:
Presumably you'd want to copy the analysis callbacks too, so the analysis results will be added to the copied experiment data, in addition to this one?

Reply:
Since we don't copy figures and analysis results with this method I figured it was best to also not copy the analysis futures which generate these.